src/bsk201903/connectMultipleClusters.md

```diff
@@ -14,14 +14,41 @@ _Should you create a single large cluster that spans multiple regions over a uni
 
 _Or should you have a lot of smaller clusters and find a way to manage and synchronise them?_
 
-Creating a single cluster that uses a unified network isn't recommended due to challenges such as:
+## A cluster to rule them all
```

Comment:
this diff does not cover the lines above, so I will put my comments on those here:

- line 5: suggested wording:
  > It's relatively common to see ~~the~~ infrastructure being replicated and distributed in different geographical regions, particularly in regulated environments.

- line 7: the requirement to handle cluster fail over is to provision headspace for fail over across clusters. The Google SRE book covers these use cases for failure isolation and mechanisms to handle cascading failures

- line 9: I would give a bit more details why ppl would want to run across regions - the articles you link actually highlight use cases [Argo use case](https://admiralty.io/blog/running-argo-workflows-across-multiple-kubernetes-clusters/) - which I feel could be at the top to motivate people to read this article. i.e. data locality or [no global locks on cluster operations](https://medium.com/@olemarkus/kubernetes-clustermesh-with-cilium-and-kops-7a8670551725)

I'm still reviewing - will add comments later as I have to run now